content/guide/dsl-spec-plugins.md

```diff
@@ -31,34 +31,59 @@ plugins:
+package_name         | no          | string      | Managed plugin package name. See [Managed Plugins](#managed-plugins). (Supported since: [cloudify_dsl_1_2](dsl-spec-versioning.html)) If `install` is `false`, `pacakge_name` is redundant. If `install` is true, `package_name` (or `source`) is mandatory.
+package_version      | no          | string      | Managed plugin pacakge version. See [Managed Plugins](#managed-plugins). (Supported since: [cloudify_dsl_1_2](dsl-spec-versioning.html))
+supported_platform   | no          | string      | Managed plugin supported platform. See [Managed Plugins](#managed-plugins). (Supported since: [cloudify_dsl_1_2](dsl-spec-versioning.html))
+distribution         | no          | string      | Managed plugin distribution. See [Managed Plugins](#managed-plugins). (Supported since: [cloudify_dsl_1_2](dsl-spec-versioning.html))
+distribution_version | no          | string      | Managed plugin distribution version. See [Managed Plugins](#managed-plugins). (Supported since: [cloudify_dsl_1_2](dsl-spec-versioning.html))
+distribution_release | no          | string      | Managed plugin distribution release. See [Managed Plugins](#managed-plugins). (Supported since: [cloudify_dsl_1_2](dsl-spec-versioning.html))
 
 <br>
 
-Example:
+# Installation Configuration
 
-{{< gsHighlight  yaml >}}
-tosca_definitions_version: cloudify_dsl_1_1
+When a plugin definition is configured with `install: true` (which is the default), `source` or `package_name` must be specified as well.
+If `pacakge_name` is specified, the manager is queried for a matching managed plugin. If one is found, it will be installed.
+If `package_name` is not specified or no matching managed plugin is found, `source` is used.
+If no managed plugin is found and `source` is not defined, plugin installation will fail.
+
+## Source Plugins
+`source` specifies where the plugin to be installed is located. Can be:
+
+* A URL to an archive of the plugin to be installed.
+* A name of a directory containing the plugin which is expected to be insde the blueprint's `plugins` directory.
+
+## Managed Plugins
+
+`package_name` specifies the name of the managed plugin to be installed. `package_version`, `supported_platfrom`, `distribution`, `distribution_version` and `distribution_release`
+may be used to explicity specify the managed plugin to install. Otherwise, an implicit resolution mechanism is employed that will fetch the latest macthing managed plugin, if one exists.
+
+TBD - When managed plugins documentation is done, link to it from here and vice versa.
+
+# Executor
 
-imports:
-    - {{page.openstack_plugin_yaml_link}}
+`executor` specifies where should the plugin be installed and in turn, where should operations using this plugin be executed. Valid values are `central_deployment_agent`
```

Comment:
a bit more clear: specifies where the plugin should be installed and in turn, where operations using this plugin be executed.